crates/dpapi-fuzzing/src/oracles/round_trip.rs

```diff
@@ -0,0 +1,13 @@
+use dpapi_core::{ReadCursor, WriteBuf};
+
+use crate::generator::AnyStruct;
+
+pub fn round_trip(any: AnyStruct) {
+    let mut buf = WriteBuf::new();
+
+    if let Ok(name) = any.encode(&mut buf) {
```

Comment:
question: By the way, we have structs that can't be encoded without it being a bug?